pkg/common/filters.go

```diff
@@ -235,14 +240,24 @@ func customizeField(field string, entity Entity) string {
 	return field
 }
 
+func customizeEntity(field string, entity Entity) Entity {
+	// NamedEntity is considered a single object, but the metdata
+	// is stored using a different entity type.
+	if entity == NamedEntity && entityMetadataFields[field] {
```

Comment:
can you just add this check to if statement customizeField() above instead of creating a separate method for each entity type?

Reply:
`customizeField` is changing the field, but not the entity. But I can update it to do both if that's what you mean?

Reply:
oh yes sorry! ignore me then, your call either way